src/main/java/com/solvd/micro9/scooters/messaging/serde/CustomSerde.java

```diff
@@ -0,0 +1,22 @@
+package com.solvd.micro9.scooters.messaging.serde;
+
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+public class CustomSerde<T> {
```

Comment:
can be refactored to the scalar class that implementing interface and returns value of scalar, e.g.,
```java
final class StringAsInt implements Scalar<Integer> {
  private final String s;
  @Override
  public Integer value() {
//  return integer value;
  }
}
```

@eyakauleva the same can be applied here